src/Domain/BlockLogicalCoordinates.cpp

```diff
@@ -110,23 +110,27 @@ block_logical_coordinates_single_point(
   }
 
   for (size_t d = 0; d < Dim; ++d) {
```

Comment:
factor into separate commit. It looks like they might not be easy to test without the shells, though. I'm okay with that.